### ISSUE TITLE
Notes above the fold?

### ISSUE THREAD

**Issue by rjcorwin** [jywarren/plots#49]
I love the Ongoing Discussions bar.  If we want to encourage folks to contribute Notes, how about we do something similar.

We could add a new "Ongoing Research" bar
<img src=http://rjsteinert.com/github/ongoing-separate.png>

Or combine them
<img src=http://rjsteinert.com/github/ongoing-combined.png>

**Comment by jywarren** [comment]
Mixing the two isn't a bad idea. I just think the mailing list tends to be a bit invisible on the site. 

Could get kind of busy. I think we'd have to do some careful design work to get that looking good. We should experiment a bit in person soon.

**Comment by rjcorwin** [comment]
Agreed, more experimentation needed.  I'm wondering if we can find a good example in the wild.  The first thing that comes to mind is from OMG Ubuntu.
http://www.omgubuntu.co.uk/2011/12/java-to-be-removed-from-ubuntu-uninstalled-from-user-machines/
<img src=http://rjsteinert.com/github/omgubuntu.png>

**Comment by jywarren** [comment]
yeah, cool. I'm almost tempted to try running some custom code to
greyscale-out all the images, so they don't distract from main page
content. That'd probably be too much thought towards a small feature.

not to mention CPU cycles... ;-)

On Wed, Dec 21, 2011 at 11:43 AM, R.J. Steinert (rjstatic) <
reply@reply.github.com

**Comment by rjcorwin** [comment]
That's a cool idea.  We could probably generate greyscale thumbnails to be used..
